book/website/communication/citable/citable-otherscite.md

```diff
@@ -0,0 +1,70 @@
+(cm-citable-otherscite)=
+# Citing other people's Research Objects
+[![](https://img.shields.io/static/v1?label=pathway&message=Research%20Software%20Engineers&color=white)](/research-software-engineers.md)
+[![](https://img.shields.io/static/v1?label=pathway&message=Software%20Citation&color=green)](/software-citation.md)
+
+When citing research objects from other people/researchers, ensure you verify when, where, and how to properly cite those research objects.
+Below, we provide a brief explanation for each of these three scenarios.
+
+(cm-citable-otherscite-when)=
+## When to cite a research object?
+
+As a rule-of-thumb, every research object that plays a significant role in your research project should be cited in you publication or methodology paper. 
+For example, when using others software to analyse your research data, it is vital to cite and attribute it properly.
+In some cases, the contribution of a research object may be small but critical to your results.
+You should always consider about giving credit to others for their work, showing gratitude to them, and try to implement more repeatable science {cite:ps}`LaZerte2021`
+
+If you are unsure about what to cite because you doubt the contribution of a particular research object to your research, you can answer the following questions from {cite:ps}`ShouldICite`
+- Does the software ask you to cite it?
+- Did the software play a critical part in, or contributed something unique to, your research?
+- Did the software manipulate or create your data? This includes storage, visualization, and communication of your data and results.
+- Do the authors of the software rely on academic credit for funding? Look for academic institutions in email addresses, URLs.
+
+If the answer to any of these questions is yes, then you must/need to cite that research object.
+
+(cm-citable-otherscite-where)=
+## Where to cite a research object?
+
+You should directly cite research objects from others in you publication where relevant.
+For example, if specific software was used to process data, it should be cited in the section describing the processing. 
+Similarly, if all the figures in a publication are created using the same software, it may be more useful to mention that software in the main text or acknowledgements. 
```

Comment:
```suggestion
Similarly, if all the figures in a publication are created using the same software, it may be more useful to mention and cite that software in the main text or acknowledgements. 
```